packages/kbn-i18n/src/angular/directive.test.ts

```diff
@@ -82,4 +83,19 @@ describe('i18nDirective', () => {
 
     expect(element.html()).toMatchSnapshot();
   });
+
+  test('sanitizes message before inserting it to DOM', () => {
```

Comment:
__nit:__ please add 2 more tests:

1. E.g. ` i18n-default-message="Default {one} onclick=alert(1) {two} message"` and `i18n-values="{ one: '<span', two: '>Press</span>' }"`

2. And similar to existing one, but with native event handler `onclick/onmouseover` instead of angular directive - something like this.